spec/services/search/google/search_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Search::Google::SearchService, type: :service do
+  describe '#call' do
+    context 'given the search result is valid' do
+      it 'returns search page body', vcr: 'search_service/keyword_ruby' do
```

Comment:
You can name the VCR as `services/search/google/valid` to know which current file is used for which test